Flakey Test CLI: ContentTypeAPIIT.Test_Create_Then_Update_Action_Mappings

```
2024-08-20T03:40:58.9804314Z [ERROR] Tests run: 15, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 18.63 s <<< FAILURE! -- in com.dotcms.api.ContentTypeAPIIT
2024-08-20T03:40:58.9806378Z [ERROR] com.dotcms.api.ContentTypeAPIIT.Test_Create_Then_Update_Action_Mappings -- Time elapsed: 3.470 s <<< FAILURE!
2024-08-20T03:40:58.9807819Z org.opentest4j.AssertionFailedError: expected: <Modified 2!> but was: <Modified!>
2024-08-20T03:40:58.9809192Z 	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
2024-08-20T03:40:58.9811027Z 	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
2024-08-20T03:40:58.9812549Z 	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
2024-08-20T03:40:58.9813832Z 	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
2024-08-20T03:40:58.9815130Z 	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
2024-08-20T03:40:58.9816360Z 	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1141)
2024-08-20T03:40:58.9817822Z 	at com.dotcms.api.ContentTypeAPIIT.Test_Create_Then_Update_Action_Mappings(ContentTypeAPIIT.java:379)
2024-08-20T03:40:58.9819174Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2024-08-20T03:40:58.9820555Z 	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:1013)
2024-08-20T03:40:58.9822300Z 	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:827)
2024-08-20T03:40:58.9823725Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2024-08-20T03:40:58.9824812Z 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
2024-08-20T03:40:58.9825459Z
```
Workflow run Nightly: https://github.com/dotCMS/core/actions/runs/10464730879

Log: 

[5_Nightly Test _ CLI Tests.txt](https://github.com/user-attachments/files/16673902/5_Nightly.Test._.CLI.Tests.txt)

--- a/dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java
+++ b/dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java
@@ -714,8 +714,7 @@ public class ContentTypeResource implements Serializable {
 							contentTypeAPI, false);
 			final ImmutableMap.Builder<Object, Object> builderMap =
 					ImmutableMap.builder()
-							.putAll(contentTypeHelper.contentTypeToMap(
-									contentTypeAPI.find(tuple2._1.variable()), user))
+							.putAll(contentTypeHelper.contentTypeToMap(tuple2._1, user))
 							.put(MAP_KEY_WORKFLOWS,
 									this.workflowHelper.findSchemesByContentType(
 											contentType.id(), initData.getUser()))

